tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/ScalaJSClassEmitter.scala

```diff
@@ -303,6 +305,34 @@ private[emitter] final class ScalaJSClassEmitter(
     }
   }
 
+  /** Generates the creation of the static fields for a JavaScript class. */
+  private def genCreateStaticFieldsOfJSClass(tree: LinkedClass)(
+      implicit globalKnowledge: GlobalKnowledge): js.Tree = {
+    val className = tree.encodedName
+    val stats = for {
+      field @ FieldDef(true, name, ftpe, mutable) <- tree.fields
+    } yield {
+      implicit val pos = field.pos
+      val classVar = envField("c", className)
+      val select = genPropSelect(classVar, genPropertyName(name))
+      js.Assign(select, genZeroOf(ftpe))
+    }
+    js.Block(stats)(tree.pos)
+  }
+
+  /** Generates the static initializer invocation of a JavaScript class. */
+  private def genStaticInitializerOfJSClass(tree: LinkedClass)(
```

Comment:
Call this `genStaticInitializationOfJSClass` or `genStaticInitializerCallForJSClass`? It doesn't generate the static initializer itself.